crates/re_types_builder/src/codegen/python.rs

```diff
@@ -1735,6 +1739,8 @@ fn quote_clear_methods(obj: &Object) -> String {
 // --- Arrow registry code generators ---
 use arrow2::datatypes::{DataType, Field, UnionMode};
 
+use super::common::ExampleInfo;
```

Comment:
strange location for something that is needed way further up

Reply:
rust-analyzer must be trippin'